docs/deploy.md

```diff
@@ -41,53 +41,59 @@ Deploying the following structure would give you the output below.
 ```
 ─ less
   └─ apis
-     └─ chat
+     └─ demo
      └─ webhooks
   └─ sockets
      └─ realtime_chat
+  └─ topics
+     └─ user_created
+  └─ functions
+     └─ sum
+  
 ```
 
 ```bash
-[less] Building... ⚙️
-[less] Build complete ✅
-[less] Deploying... 🚀
-[less] Deployment complete ✅
-[less] Resources
-[less]   - API URLs
-[less]     - chat: https://a2m1n3.execute-api.eu-west-1.amazonaws.com
-[less]     - webhooks: https://n2s9n5.execute-api.eu-west-1.amazonaws.com
-[less]   - Web Socket URLs
-[less]     - realtime_chat: wss://10l06n.execute-api.eu-west-1.amazonaws.com
-[less] 🇨🇻
+[less-cli] Building... ⚙️
+[less-cli] Build complete ✅
+[less-cli] Deploying... 🚀
+[less-cli] Deployment complete ✅
+[less-cli] Resources
+[less-cli]   - API URLs
+[less-cli]     - chat: https://[PROJECT_NAME]-chat.api.eu-0.a83b464c9.less.chuva.cv
+[less-cli]     - webhooks: https://[PROJECT_NAME]-webhooks.api.eu-0.a83b464c9.less.chuva.cv
+[less-cli]     - topics: https://[PROJECT_NAME]-topics.api.eu-0.a83b464c9.less.chuva.cv
+[less-cli]     - functions: https://[PROJECT_NAME]-functions.api.eu-0.a83b464c9.less.chuva.cv
+[less-cli]   - Web Socket URLs
+[less-cli]     - realtime_chat: wss://[PROJECT_NAME]-realtime-chat.api.eu-0.a83b464c9.less.chuva.cv
```

Comment:
on socket  realtime_chat instead of `.api.` on the url it should be `.ws.`

Reply:
Thanks. Fixed.